.gitignore

```
@@ -1 +1,2 @@
 /.idea
+.gradle
```

```suggestion
/.gradle
```
sau khi apply xong em remove cache để nó loại bỏ các file trong folder này rồi push lên lại nhé!

ref: https://stackoverflow.com/questions/13541615/how-to-remove-files-that-are-listed-in-the-gitignore-but-still-on-the-repositor